config/manager/manager.yaml

```diff
@@ -29,6 +29,9 @@ spec:
         - /manager
         args:
         - --leader-elect
+        env:
+        - name: RELATED_IMAGE_LIMITADOR
+          value: "quay.io/3scale/limitador:latest"
```

Comment:
You mentioned the change to _Kuadrant_ repo instead of 3scale, maybe after copying the images we could include those changes here as well?

Reply:
I prefer to update all references in another PR and when the `latest` tag exists in quay.io/kuadrant/lmitador image